config/session.php

```diff
@@ -16,7 +16,7 @@
     |
     */
 
-    'driver' => 'file',
+    'driver' => env('SESSION_DRIVER', 'file'),
```

Comment:
env() changes aren't going to be merged because October doesn't use the .env file by default, users need to run `php artisan october:env` to get it.